src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,91 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.DemandType;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.team6479.lib.subsystems.TankDrive;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.DrivetrainConstants;
+
+public class Drivetrain extends SubsystemBase implements TankDrive{
+  
+  private TalonFX leftFront;
+  private TalonFX leftBack;
+  private TalonFX rightFront;
+  private TalonFX rightBack;
+
+  public Drivetrain() {
+    // construct motors
+    leftFront = new TalonFX(DrivetrainConstants.LEFT_FRONT);
+    leftBack = new TalonFX(DrivetrainConstants.LEFT_BACK);
+    rightFront = new TalonFX(DrivetrainConstants.RIGHT_FRONT);
+    rightBack = new TalonFX(DrivetrainConstants.RIGHT_BACK);
+
+    // set motors to factory defaults on startup
+    leftFront.configFactoryDefault();
+    leftBack.configFactoryDefault();
+    rightFront.configFactoryDefault();
+    rightBack.configFactoryDefault();
+
+    // left Back follows left Front and right Back follows right Front
```

Comment:
```suggestion
    // back motors follow front motors
```
Phrasing is a bit awkward.